src/main/kotlin/app/revanced/util/fingerprint/FingerprintUtils.kt

```diff
@@ -0,0 +1,21 @@
+package app.revanced.util.fingerprint
+
+import app.revanced.patcher.fingerprint.method.impl.CustomFingerprint
```

Comment:
This reference does not exist; merging this PR will cause a build error. Either don't use the typealias here yet, or push a commit to bump the version of ReVanced Patcher once the typealias is merged.